### ISSUE TITLE
confirmation requirement to transform battery into pandora

### ISSUE THREAD

**Issue by FULANO93** [PSP-Archive/ARK-4#352]
### New Feature

Well, yesterday I updated the version of ark and I was looking at the advanced options in the vsh menu to see if there was anything new and I ended up pressing the rigth button on dpad while highlighting the option to transform the battery into pandora, the menu just closed and I didn't think it was anything to worry about...
I played all night and turned off the PSP... today when I went to turn it on, I found it stuck on a black screen and not activating lights of the card... I thought I had bricked it somehow...
I turned it off by holding the power button and tried to turn it on in recovery mode by holding the R button, it didn't work...
I tried to remove the battery and put it back in and it was then that the PSP lit up by itself when connecting the battery and I realized that I could have accidentally turned the battery into a Pandora...
I plugged it into the charger without using the battery and then the PSP turned on normally...
I connected the battery right away and went to the vhs menu, I saw that the option was `pandora > normal`...
I pressed to right on dpad highlighting that option and the menu closed without confirming anything or any warning...
I went to the vs menu again and now the option was `normal > pandora`

In short, it would be good if they put a confirmation or a warning when clicking to transform the battery into pandora, or a protection, like the option to protect the usb and flash0, remove the confirmation by pressing right on the dpad.
